src/main/java/com/home/qa/base/TestBase.java

```diff
@@ -6,7 +6,6 @@
 import java.util.concurrent.TimeUnit;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
```

Comment:
Why have this been removed?
Are you sure it's still working without the import?

Reply:
sorry I see it's working.